app/services/SpellcheckService.java

```diff
@@ -1,42 +1,73 @@
 package services;
 
 import com.google.inject.Inject;
-import dk.dren.hunspell.Hunspell;
 import lombok.val;
 import org.apache.commons.lang3.StringUtils;
+import org.languagetool.JLanguageTool;
+import org.languagetool.language.BritishEnglish;
+import org.languagetool.rules.RuleMatch;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.FileNotFoundException;
-import java.io.UnsupportedEncodingException;
-import java.util.Arrays;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
 import java.util.stream.Collectors;
 
 public class SpellcheckService {
 
-    private Hunspell.Dictionary dictionary;
+    private BritishEnglish britishEnglish;
     private final static Logger LOGGER = LoggerFactory.getLogger(SpellcheckService.class);
 
     @Inject
     public SpellcheckService() {
-        dictionary = loadDictionaries();
+        britishEnglish = new BritishEnglish();
     }
 
-    public String getSpellcheckSuggestionsString(String[] wordsToCheck) {
+    public String getSpellCheckSuggestions(String textToSpellCheck) {
         StringBuilder spellingMistakes = new StringBuilder();
-
-        Arrays.stream(wordsToCheck).forEach(word -> {
-                if (dictionary.misspelled(word)) {
-                    spellingMistakes.append("\"").append(word).append("\" : ");
```

Comment:
Probably worth taking a look after this is merged to see how we could use Jackson to render JSON